src/Zynga/Framework/IO/Web/V1/Manager.hh

```diff
@@ -50,4 +53,25 @@ class Manager {
       throw new UnexpectedHttpCodeException((string)$returnCode);
     }
   }
+  
+  public static function makeCurlRequest(
+    UrlBox $uploadUrl,
+    Map<string, mixed> $postParams): array<string, mixed> {
```

Comment:
Why not make a model for these CURL parameters?

Reply:
We can't have a strictly defined key names and strictly defined value types. It would be an overkill to create a model which just has Map<string, mixed> $postParams